### PR TITLE
New schema

### PR DESCRIPTION
**Pull request by tweddielin**
Main changes are in the big-model-graph page, where now tyra pull up the top models from results.ranked_table (cache table) for each model_group_id aggregated by averaging over time. Also, the layout is changed with a dropdown model_comment menu. 

- Change as_of_date with evaluation_start_time in query, but in react component, props are still asOfDate and setAsOfDate. They might be changed to evaluationDate and setEvaluationDate in later PR.
- Now the model-selectgraph component takes new endpoint ```get_model_groups(model_comment)``` as ajax call using the new query function ```get_model_groups()```.
- Implement couple data processing codes to fit the Highchart format by functional programming. 
- The multiple parameter-metric selection is removed. We can only choose one parameter-metric at a time.
- Made some changes in main.css.

### CODE REVIEW

**Comment by tweddielin** [comment]
close issue #48

**Comment by tweddielin** [comment]
close issue #44

**Comment by tweddielin** [comment]
close #49

File: webapp/query.py [hunk 1]
```diff
@@ -43,6 +43,64 @@ def get_model_prediction(query_arg):
     return output
 
 
+def get_model_comments(run_time):
```
Comment (thcrock) [inline webapp/query.py]: I see that this method exists but isn't used. Is it tough to get this to seed the list of model comments on the frontend?

**Pull request merged**